service/history/timerQueueProcessor.go

```diff
@@ -292,6 +304,7 @@ func (t *timerQueueProcessorImpl) internalProcessor(tasksCh chan<- *persistence.
 		}
 
 		if isWokeByNewTimer {
+			t.metricsClient.IncCounter(metrics.TimerQueueProcessorScope, metrics.NewTimerNotifyCounter)
```

Comment:
i wanted to see if this has any correlation with get timer queries. would be helpful for debugging.